infrateam_first_responder.md

```diff
@@ -85,6 +86,20 @@ Status check from the server (ssh into the prod server for that project and then
 - Workflow server rails: `curl -i https://workflow-service-prod.stanford.edu/status/all`
 - Suri: `curl -i https://sul-suri-prod.stanford.edu/status/all`
 
+### Run infrastructure-integration-tests
+
+We want the FR to ensure
+  - dependency updates don't break cross-app functionality
+  - this test suite remains useful
+
+To that end, all the infrastructure-integration-tests must be run successfully in the qa and/or stage environment before deploying to production (one of qa or stage is sufficient;  testing in environments helps ensure our environments are (still) set up properly.).  This should be done as part of deployment of dependency updates as described in the next section.
+
+If some tests fail when running the whole test suite at once, but pass when run individually, that is ok -- as long as they each pass under some circumstances. Chances are not great that the full suite will pass in one go from a spotty off-campus connection.
+
+If you're unsure whether a particular test failure indicates bad network luck, a regression in the application, or an out of date test, raise it for discussion (or ask another dev to retry from their laptop) in #dlss-infrastructure.
+
+If on a Mac, you will get better results if you stay in the same "space" as the running tests (avoids focus issues with the browser).
```

Comment:
Does this belong here or in infra-integration test README?

Reply:
This was here previously and I didn't add it. I'm guessing @ndushay did, and I'm happy to defer that question to her.

Reply:
In which case don't worry about it.

Reply:
yes, it would be better in infra-integration-tests README.  But I don't have cycles for that right now.